### DISCUSSION TITLE
Question: How to move marker?

### DISCUSSION THREAD

**Discussion by lukepower** [inocan-group/vue3-google-map#224] [Q&A]
Hi, 
is there any way to move an existing marker to other coordinates?
When changing the underlying data the marker is not updated, and there seems to be no way to call the API's setPosition function...

Thanks

**Comment by HusamElbashir** [comment]
Yes there is you'll just have to replace the entire `position` object. Unfortunately we don't deep watch options so you can't just change `lat`/`lng`.

**Comment by nonickDev** [comment]
> Hi, is there any way to move an existing marker to other coordinates? When changing the underlying data the marker is not updated, and there seems to be no way to call the API's setPosition function...
> 
> Thanks

`<Marker :options="{ position: {lat: lat, lng: lng}, draggable: true }" @dragend="markerDragend"/>`

`methods: {
    markerDragend(event) {
      console.log('dragend', event.latLng.lat(), event.latLng.lng())
    },
}`
